Tasks/Kachura/Task3_kirillkachura.py

```diff
@@ -0,0 +1,95 @@
+from ast import Break, Continue
+
+
+def spaces (x):
+    if len(x) <= str_length:
+        x = x.replace(' ', '  ', str_length - len(x))
+    if len(x) <= str_length:
+        x = x.replace('  ', '   ', str_length - len(x))
+    if len(x) <= str_length:
+        x = x.replace('   ', '    ', str_length - len(x))
+    if len(x) <= str_length:
+        x = x.replace('    ', '     ', str_length - len(x))
+    if len(x) <= str_length:
+        x = x.replace('     ', '      ', str_length - len(x))
```

Comment:
you're sure that's enough? maybe a couple of those checks more will do the trick? -1 point for the unwise choice